docs/en/modules/net.md

```diff
@@ -188,8 +188,8 @@ sk = net.createConnection(net.TCP, 0)
 sk:on("receive", function(sck, c) print(c) end )
 sk:connect(80,"192.168.0.66")
 sk:on("connection", function(sck,c)
-    -- Wait for connection before sending.
-    sk:send("GET / HTTP/1.1\r\nHost: 192.168.0.66\r\nConnection: keep-alive\r\nAccept: */*\r\n\r\n")
+  -- Wait for connection before sending.
+  sk:send("GET / HTTP/1.1\r\nHost: 192.168.0.66\r\nConnection: keep-alive\r\nAccept: */*\r\n\r\n")
```

Comment:
Ahh right, had wanted to fix this for a long time but forgot again :frowning: Thanks, I'll fix it.